src/nostr-profile/nostr-profile.ts

```
@@ -0,0 +1,819 @@
+  private isStatsError: boolean = false; 
+
+  private stats = {
+  follows: 0,
+  followers: 0,
+  notes: 0,
+  replies: 0,
+  zaps: 0,
+  relays: 0,
+  };
+
+  private onClick: Function | null = null;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)